### ISSUE TITLE
Incompatibility with conda 23.5.0 ?

### ISSUE THREAD

**Issue by noamgat** [mamba-org/mamba#2547]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### How did you install Mamba?

conda install mamba

### Search tried in issue tracker

conda 23.5 + python 3.11 incompatibility?

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

I do not have this problem with Conda, just with Mamba

### Describe your issue

We use mamba in our CI/CD pipelines. 
We start from the continuum anaconda docker image (for other reasons). The installation happens using the line

```conda install mamba ```

Afterwards we run

```mamba env create -f environment.yml```

And have started getting the following error in runs in the last 2-3 days despite not changing anything (see the logs).
Forcing the conda version to 23.3.1 (previous version released) fixes the issue.
Conda 23.5.0 was released a few days ago as well, raising my suspicion.

### Logs

```shell
Traceback (most recent call last):
  File "/opt/conda/bin/mamba", line 7, in <module>
    from mamba.mamba import main
  File "/opt/conda/lib/python3.10/site-packages/mamba/mamba.py", line 49, in <module>
    import libmambapy as api
  File "/opt/conda/lib/python3.10/site-packages/libmambapy/__init__.py", line 7, in <module>
    raise e
  File "/opt/conda/lib/python3.10/site-packages/libmambapy/__init__.py", line 4, in <module>
    from libmambapy.bindings import *  # noqa: F401,F403
ImportError: libarchive.so.13: cannot open shared object file: No such file or directory
Cleaning up project directory and file based variables
```

**Comment by jonashaag** [comment]
Starting from Anaconda is not recommended, it sometimes causes problems, see

https://github.com/mamba-org/mamba/issues/1826
https://github.com/mamba-org/mamba/issues/1728
https://github.com/mamba-org/mamba/issues/1724

**Comment by itamarst** [comment]
I was able to work around this by installing mamba like so:

`conda install --override-channels -c conda-forge mamba`

This ensures the libarchive comes from conda-forge.

**Comment by itamarst** [comment]
I also had issues with PyPy being installed, so ended up with this:

`conda install --override-channels -c conda-forge mamba 'python_abi=*=*cp*'`

Perhaps this should be the recommended installation command in the docs.

**Comment by jonashaag** [comment]
Agreed, that's much better than mixing channels! Does it break anything that's shipped with the Anaconda distribution and uses "their" libarchive though?

**Comment by itamarst** [comment]
I'm doing this in a Docker image. As long as my packaging tests pass I'm assuming it's fine, but it's not exercising every edge case a user might have... Possibly I should consider switching to micromamba at some point.

For what it's worth my general feeling is that:

1. Mamba should be installable in default Conda environment; telling people to switch to miniforge means far fewer users.
2. Therefore, Mamba may wish to switch to vendoring all the things in this situation. This is obviously very much against the Conda packaging model, but it seems reasonable in this situation (much like `pip` vendors its dependencies).

**Comment by jonashaag** [comment]
I think we should adopt your workaround temporarily.

Long term we will probably stop development on Mamba and focus on Micromamba.

**Comment by SilentGene** [comment]
I also encountered this and have solved it by installing the `libarchive13` lib on my Ubuntu:
`$ sudo apt install libarchive13`

**Comment by jonashaag** [comment]
That might work if you're lucky but it's not recommended since the libarchive versions might be incompatible

**Comment by jjerphan** [comment]
This problem as been fixed by mamba and micromamba 2.0 since they do not dépend on conda and do not use `defaults` by default anymore.
